Input/Input.css

```diff
@@ -0,0 +1,58 @@
+body {
+    font-family: "Roboto", sans-serif;
+    font-size: 0.8rem;
+  }
+  body {
+    font-family: "Roboto", sans-serif;
+    font-size: 0.8rem;
+  }
+  .wrapper {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+  
+  .input-container {
+    align-items: center;
+    background-color: #f0f0f0;
+    border-radius: 0.3rem;
+    border: 1px solid black;
+    display: flex;
+    flex-direction: column;
+    gap: 0.8rem;
+    justify-content: center;
+    padding: 2rem;
+    width: 600px;
+  }
+  .input-text {
+    padding: 0.5rem 3rem;
+    border: 1.5px solid black;
+  }
+  
+  .input-error {
+    padding: 0.5rem 3rem;
+    border: 1.5px solid red;
+    color: red;
+  }
+  .label-input-error {
+    color: red;
+  }
+  .error-message {
+    padding: 0.5rem 4rem;
+    color: red;
+  }
+  
+  .input-success {
+    padding: 0.5rem 3rem;
+    border: 1.5px solid rgb(60, 170, 60);
+    color: rgb(60, 170, 60);
```

Comment:
instead of writing the same colour, again and again, you can set it as a variable then use it anywhere.